x-pack/legacy/plugins/maps/public/layers/util/can_skip_fetch.js

```diff
@@ -168,5 +168,5 @@ export function canSkipFormattersUpdate({ prevDataRequest, nextMeta }) {
     return false;
   }
 
-  return !_.isEqual(prevMeta.fieldNames, nextMeta.fieldNames);
+  return _.isEqual(prevMeta.fieldNames, nextMeta.fieldNames);
```

Comment:
I found a bug in this method will working on this PR. The statement was inverted